libcxx/test/std/algorithms/alg.nonmodifying/alg.find/find.pass.cpp

```diff
@@ -215,7 +217,9 @@ TEST_CONSTEXPR_CXX20 bool test() {
   Test<TriviallyComparable<wchar_t>, TriviallyComparable<wchar_t>>().operator()<TriviallyComparable<wchar_t>*>();
 #endif
 
-#if TEST_STD_VER >= 20
+  // TODO: Remove the `_LIBCPP_ENABLE_EXPERIMENTAL` check once we have the FTM guarded or views::join isn't
+  // experimental anymore
+#if TEST_STD_VER >= 20 && (!defined(_LIBCPP_VERSION) || defined(_LIBCPP_ENABLE_EXPERIMENTAL))
```

Comment:
This looks like a bad merge conflict resolution with a recent patch I did.

```suggestion
#if TEST_STD_VER >= 20
```